aws_lambda_builders/workflows/python_pip/actions.py

```diff
@@ -35,8 +35,8 @@ def execute(self):
         try:
             package_builder.build_dependencies(
                 self.artifacts_dir,
```

Comment:
While we are here, could you add kwarg mapping when calling build_dependencies here?

Reply:
Pushed up new commit with these changes